Profiles - ZIB 2017/zib-IllnessPerception.xml

```diff
@@ -110,16 +110,10 @@
         <coding>
           <system value="http://snomed.info/sct" />
           <code value="27026000" />
+          <display value="Insight"/> 
```

Comment:
Adding a required display value to profiles is undesired (and out of scope for this ticket). Please revert this change.

Reply:
Fix!